highlighting/generate-rules-highlighting.py

```diff
@@ -80,13 +80,13 @@ def get_Traduction(KEYWORDS_PATH, KEYWORDS_PATTERN):
                 tmp[languageCode][k] = str(yamlFile[k])
 
     Result = {}
```

Comment:
I know this is present in a lot of you code already, but could you start sticking with the convention that variables start with lower case letters, and use snake_case (so for example `result` and `language_code`)

Reply:
It's done, for all variable (except constant)